php/WP_CLI/CommandWithTranslation.php

```diff
@@ -269,6 +267,45 @@ public function activate( $args, $assoc_args ) {
 	}
 
 	/**
+	 * Download a language pack.
+	 *
+	 * @see wp_download_language_pack()
+	 *
+	 * @param string $download Language code to download.
+	 * @return string|WP_Error Returns the language code if successfully downloaded, or a WP_Error object on failure.
+	 */
+	protected function download_language_pack( $download ) {
```

Comment:
Can we make this `private`? I'd like to prevent subclassing so we don't have to worry about backwards-compatibility if we decide to ever change things.